extensions/amp-story-interactive/0.1/amp-story-interactive-slider.js

```diff
@@ -99,7 +99,7 @@ export class AmpStoryInteractiveSlider extends AmpStoryInteractive {
   onDrag_() {
     const {value} = this.inputEl_;
     this.bubbleEl_.textContent = value + '%';
-    this.bubbleEl_.classList.add('show');
+    this.rootEl_.classList.add('show');
```

Comment:
Let's make this class called with `i-amphtml-story-interactive-slider-show`

Reply:
I wonder if `-interacting` of `-moving` would be descriptive of the user action.